server/src/main/java/com/hedera/block/server/BlockNodeApp.java

```diff
@@ -52,13 +54,15 @@ public class BlockNodeApp {
     @Inject
     public BlockNodeApp(
             @NonNull final ServiceStatus serviceStatus,
+            @NonNull final WebServerStatus webServerStatus,
             @NonNull final HealthService healthService,
             @NonNull final PbjBlockStreamService pbjBlockStreamService,
             @NonNull final PbjBlockAccessService pbjBlockAccessService,
             @NonNull final WebServerConfig.Builder webServerBuilder,
             @NonNull final ServerConfig serverConfig,
             @NonNull final ConfigurationLogging configurationLogging) {
         this.serviceStatus = requireNonNull(serviceStatus);
+        this.webServerStatus = webServerStatus;
```

Comment:
```suggestion
        this.webServerStatus = requireNonNull(webServerStatus);
```

In general we use preconditions on constructors to fail fast.

Reply:
Updated